microsoft-edge/webview2/index.md

```diff
@@ -167,6 +167,14 @@ You may run WebView2 apps on the following versions of Windows.
 > [!IMPORTANT]
 > \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems][DeployedgeMicrosoftEdgeSupportedOS].  
 
+WebView2 also works in the following Windows Embedded releases
+
+*   Windows IoT Enterprise LTSC x32
+*   Windows IoT Enterprise LTSC x64
+*   Windows IoT Enterprise 21h1 x64
+
+Windows IoT Enterprise on Arm64 is currently under investigation.
```

Comment:
@jasonstephen15  - may we drop this one until we know for sure?

Reply:
We actually wanted that in there to show we are looking at it (it should work but we need to verify).